### DISCUSSION TITLE
2 factor inability to add phone number problem

### DISCUSSION THREAD

**Discussion by HadronSecurity** [community/community#76642] [Other Feature Feedback, Questions, & Ideas]
Merhaba Ben bu gün github hesabıma girdim fakat bana üste bahsetdiği gibi dediki 2 faktörlü doğrulmayı aç dedi bende açtım dedimki telefon numarası ekleyim dedim ve bi baktım telefon numaralarının arasında türkiye yok malesef bulamadım baktım saatlerce gine yok bu sorunu nasıl çözebilirim.
![Ekran görüntüsü 2023-11-22 205854](https://github.com/community/community/assets/147801258/602a9f89-70af-4cb7-b278-b448380524f3)

**Comment by lotirium** [comment]
A similar issue was already reported on GitHub's community discussions. The user managed to work around this problem by manually adding the country code through the frontend using the inspect feature of their browser, and it worked well for them. You can check it out here
https://github.com/orgs/community/discussions/32099

**Comment by samus-aran** [comment]
Hi @HadronSecurity ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
